src/outro.js

```diff
@@ -1,2 +1,13 @@
+for (var key in MathQuill) (function(key, val) {
+  if (preInterVerMathQuill[key]) return; // already set .noConflict
+  if (typeof val === 'function') {
+    preInterVerMathQuill[key] = function() {
+      insistOnInterVer();
+      return val.apply(this, arguments);
+    };
+    preInterVerMathQuill[key].prototype = val.prototype;
+  }
+  else preInterVerMathQuill = val;
```

Comment:
This line is in error, see my comment above.  It should be:
```javascript
else preInterVerMathQuill[key] = val;
```

We will be updating our MathQuill version ASAP.  I will try to get a pull request soon.